core/domain/feedback_validators.py

```diff
@@ -152,6 +153,25 @@ def _get_external_id_relationships(cls, item):
             )
         return field_name_to_external_model_references
 
+    @classmethod
+    def _validate_text(cls, item):
+        """Validate text length is less than 10000 characters.
```

Comment:
Nit: Can we make this third-person, i.e. "Validates"...

```suggestion
        """Validates text length is less than 10000 characters.
```

Reply:
Done.